universal-login-sdk/lib/sdk.ts

```diff
@@ -40,6 +43,15 @@ class UniversalLoginSDK {
     return [privateKey, contract.address];
   }
 
+  async getFutureWallet() {
+    const {address, privateKey} = Wallet.createRandom();
+    this.config = this.config || (await this.getRelayerConfig()).config;
```

Comment:
getRelayerConfig() {
    return this.config = this.config || (await this.doGetRelayerConfig()).config;
}